steward/src/commands/cosmos_to_eth.rs

```diff
@@ -10,12 +10,13 @@ use std::{process::exit, time::Duration};
 
 const TIMEOUT: Duration = Duration::from_secs(60);
 
-/// Send Cosmos to the Eth chain
+/// Send Cosmos to the Eth chain. args is a Vector of Strings and flag_no_batch is boolean
```

Comment:
We shouldn't need to include this typing information in the command-level documentation when the flag details should explain it below.